include/dxc/Support/SPIRVOptions.h

```diff
@@ -84,6 +84,7 @@ struct SpirvCodeGenOptions {
   llvm::SmallVector<llvm::StringRef, 4> optConfig;
   std::vector<std::string> bindRegister;
   std::vector<std::string> bindGlobals;
+  llvm::StringRef entrypointName;
```

Comment:
I am worried that later this `llvm::StringRef` loses the actual string object i.e., dangling pointer.
I am surprised other members here use `llvm::StringRef`.

If there is no strong guarantee that it will not be a dangling pointer, could you please change it to `std::string`?

Reply:
Done